eds/interfaces/worker.py

```diff
@@ -22,3 +24,8 @@ def build_eds(self, eds_version: bool, plugin_versions: bool) -> None:
             plugin_versions (bool): List of EDS plugin versions as pip install requirements.
         """
         pass
+
+
+PluginInterface.register(Worker)
+assert issubclass(Worker, PluginInterface)
+assert isinstance(Worker(), PluginInterface)
```

Comment:
all of the `isintance` assertions are failing in the interface classes.  those assertions should fail.  they shouldn't be in here.

Reply:
Yeah I can remove these (or we can move work to the new issue #25?)